Makefile

```diff
@@ -102,21 +102,28 @@ compile_commands.json:
 
 
 build-debug: CONFIG = debug
-build-debug: SWIFTBFLAGS = -Xswiftc -target -Xswiftc x86_64-apple-macosx10.13 --configuration $(CONFIG)
-build-release: BAZELFLAGS = --announce_rc
+build-debug: SWIFTBFLAGS = -Xswiftc -target \
+	-Xswiftc x86_64-apple-macosx10.13 \
+	--configuration $(CONFIG)
+build-debug: BAZELFLAGS = --announce_rc \
+	--disk_cache=$(HOME)/Library/Caches/Bazel
 build-debug: build-impl
 
 build-release: CONFIG = release
-build-release: SWIFTBFLAGS = -Xswiftc -target -Xswiftc x86_64-apple-macosx10.13 --configuration $(CONFIG) -Xswiftc -static-stdlib
-build-release: BAZELFLAGS = --announce_rc --compilation_mode opt
+build-release: SWIFTBFLAGS = -Xswiftc -target \
+	-Xswiftc x86_64-apple-macosx10.13 \
+	--configuration $(CONFIG) -Xswiftc -static-stdlib
+build-release: BAZELFLAGS = --announce_rc \
+	--compilation_mode opt \
+	--disk_cache=$(HOME)/Library/Caches/Bazel
```

Comment:
Tangent - our SPM builds use Whole module optimization, which should be here too if `opt` doesn't set that.

Reply:
The program is fundamentally faster with WMO - when comparing our SPM -> Bazel builds. I'll submit a followup to add this, which will be useful for when we want to build releases with Bazel

Reply:
👍 - will address in a followup